arm-network/2016-09-01/swagger/network.json

```diff
@@ -4297,6 +4347,76 @@
     }
   },
   "definitions": {
+	  "ApplicationGatewayBackendHealth": {
+	    "properties": {
+	      "BackendAddressPools": {
+	        "type": "array",
+	        "items": {
+	          "$ref": "#/definitions/ApplicationGatewayBackendHealthPool"
+	        }
+	      }
+	    },
+	    "description": "List of backendhealth pools."
+	  },	
+	  "ApplicationGatewayBackendHealthPool": {
+      "properties": {
+        "BackendAddressPool": {
```

Comment:
General comment: model properties of all the new definitions added in this pull request are in PascalCase, for example:
- "BackendAddressPool"
- "BackendHttpSettingsCollection"

All the other model definitions have the model properties in camelCase. This looks weird. Can you please double check the fiddler response and make sure that this is correct.